docs/cloud/how-to-export-workflow-event-histories.md

```diff
@@ -0,0 +1,194 @@
+
+For example:
+
+```command
+tcld namespace export create --namespace "your-namespace.your-account" --sink-name "your-sink-name" --role-arn "arn:aws:iam::123456789012:role/test-sink" --s3-bucket-name "your-aws-s3-bucket-name”
+```
+
+Retrieve the status of this command by running the `tcld namespace export get` following command.
+
+For example:
+
+```command
+tcld namespace export get --namespace "your-namespace.your-account" --sink-name "your-sink-name"
+```
+
+The following is an example of the output:
+
+```json
+{
+  "name": "your-sink-name",
+  "resourceVersion": "a6442895-1c07-4da4-aaca-58d57d338345",
+  "state": "Active",
+  "spec": {
+    "name": "your-sink-name",
+    "enabled": true,
+    "destinationType": "S3",
+    "s3Sink": {
+      "roleName": "your-export-test",
+      "bucketName": "your-export-test",
+      "region": "us-east-1",
+      "kmsArn": "",
+      "awsAccountId": "123456789012"
+    }
+  },
+  "health": "Ok",
+  "errorMessage": "",
+  "latestDataExportTime": "0001-01-01T00:00:00Z",
+  "lastHealthCheckTime": "2023-08-14T21:30:02Z"
+}
+```
+
+## Monitor export progress {#monitor}
+
+**How to monitor the History exportation progress?**
+
+Once you've finalized the setup, here's how to monitor the export progress:
+
+1. **Export Job Execution**:
+   - **Schedule**: The Export job is scheduled to run on an hourly basis, starting at 10 minutes past each hour.
+     This ensures that the history data of Closed Workflows is exported to your designated S3 bucket approximately 60 minutes post Workflow closure.
+   - **Duration**: The time taken for the export process can vary based on the amount of data, so it may not be instantaneous.
+     Be patient and check the S3 bucket after the scheduled time.
+2. **Checking the S3 Bucket**:
+   - **Files Arrival**: Post the initial hour of setting up, inspect your S3 bucket.
+     You should see the exported Workflow History files.
+     These files encapsulate the relevant data from the Closed Workflows.
+   - **Directory Structure**: Your exported files will adhere to the following naming convention and path:
+
+   ```command
+   s3://[bucket-name]/temporal-workflow-history/export/[Namespace]/[Year]/[Month]/[Day]/[Hour]/[Minute]/
+   ```
+
+3. **UI Insights**:
+   - **Last Successful Export**: This displays the timestamp of the most recent successful export.
+     It's an essential indicator of the last time your export process completed without any hitches.
+   - **Last Status Check**: This reflects the timestamp of the latest internal Workflow check.
+     This internal check routinely evaluates the health and status of the Export mechanism, ensuring its uninterrupted functioning.
+
+For optimal results, make it a habit to frequently review the S3 bucket for any new exported files and consistently refer to the UI insights.
+This dual check ensures you remain abreast of the export progress and any potential issues.
+
+## Verify export setup {#verify}
```

Comment:
Can we reference "verify export setup" on line 85, in the "automated setup"?

We suggest user to click "verify" before they click save. So they could find any configuration setting earlier, don't need to wait for the whole hour.

Reply:
Added